### PR TITLE
Extra spacing is added after each <hottext> element in tokenhighlight

### PR DESCRIPTION
**Pull request by pooja31chaudhary**
Extra spacing is added after each <hottext> element in tokenhighlight. 
Fixed.

### CODE REVIEW

**Comment by jack-lin-lrn** [comment]
LGTM

File: src/Processors/QtiV2/In/Interactions/HottextInteractionMapper.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ private function buildTemplate(HottextInteraction $interaction)
 
             // Inject token spans where there are inline hottext nodes
             // HACK: because these tokens are inline, make sure there is space between the token and its inline siblings
-            $content = str_replace($hottextString, ' '.QtiMarshallerUtil::marshall($tokenSpan).' ', $content);
+            $content = str_replace($hottextString, ' '.QtiMarshallerUtil::marshall($tokenSpan).'', $content);
```
Comment (jack-lin-lrn) [inline src/Processors/QtiV2/In/Interactions/HottextInteractionMapper.php]: Could you please remove `.''` and add spaces for `.`?
```
$content = str_replace($hottextString, ' ' . QtiMarshallerUtil::marshall($tokenSpan), $content);
```

Comment (pooja31chaudhary) [reply src/Processors/QtiV2/In/Interactions/HottextInteractionMapper.php]: Done

Comment (ttton) [reply src/Processors/QtiV2/In/Interactions/HottextInteractionMapper.php]: Just checking in on this one @pooja31chaudhary, I think you may have misunderstood the suggestion. Is there a reason for the trailing appended empty string? Can it just be removed? Also I've noticed tab indentation creeping back in, wondering if you can fix that up quickly.

Comment (pooja31chaudhary) [reply src/Processors/QtiV2/In/Interactions/HottextInteractionMapper.php]: Done

**Pull request merged**